### DISCUSSION TITLE
Getting error with @scullyio

### DISCUSSION THREAD

**Discussion by piuma8** [scullyio/scully#1545] [General]
I am trying to add @scullyio to my app, but I always get this error: 
$ ng add @scullyio/init
ℹ Using package manager: npm
✔ Found compatible package version: @scullyio/init@2.1.14.
✔ Package information loaded.

The package @scullyio/init@2.1.14 will be installed and executed.
Would you like to proceed? Yes
✔ Package successfully installed.
? Which route renderer would you like to use? Scully platform server
    Installing ng-lib
    Installing Scully Platform Server plugin
UPDATE src/app/app.module.ts (683 bytes)
UPDATE src/polyfills.ts (2531 bytes)
UPDATE package.json (1280 bytes)
⠙ Installing packages (npm)...npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree 
npm ERR!
npm ERR! While resolving: my-blog@0.0.0
npm ERR! Found: @angular/animations@13.0.3
npm ERR! node_modules/@angular/animations
npm ERR!   @angular/animations@"~13.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/animations@"12.2.16" from @angular/platform-server@12.2.16       
npm ERR! node_modules/@angular/platform-server
npm ERR!   @angular/platform-server@"^12" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See C:\Users\Roberto\AppData\Local\npm-cache\eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Roberto\AppData\Local\npm-cache\_logs\2022-02-21T15_01_18_296Z-debug.log
✖ Package install failed, see above.
The Schematic workflow failed. See above.

I am trying installing it manually, but cannot solve. Someone can help me, please? Thank you!

**Comment by piuma8** [comment]
After launching npm install --legacy-peer-deps now is working, but what was the problem?

**Reply by SanderElias** [reply]
NPM is not handling the deps in the best way. We are still looking for a workaround. In our V3 branch this problem is probably gone.
